site/layouts/partials/footer.html

```diff
@@ -44,5 +44,58 @@
     </script>
     {{ end }}
 
+    <script>
+    document.addEventListener('DOMContentLoaded', async () => {
+        const elems = document.querySelectorAll(".krew-plugin-count");
+        if (elems.length === 0){
+            return;
+        }
+        let count = '<error>';
+        try {
+            const resp = await fetch('/.netlify/functions/api/pluginCount');
+            const json = await resp.json();
+            count = json.data?.count ?? '<error>';
```

Comment:
It seems like optional chaining is relatively [new](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#Browser_compatibility) (or at least those version numbers seem a little high). Do either of you think this should be changed to the old way or is it ok?

Reply:
The question immediately becomes: how far back do we want to be backwards compatible.

This case is trivial, but `import` and `async/await` is already more involved and the old style quickly becomes kind of ugly. We could warn page visitors by including a `<script nomodule ..>` script to inform the user that the page is not working as intended and to switch to a newer browser.

Reply:
I think these are mostly fine. I tend to assume developers use latest browser versions most of the time. In that regard they're more savvy than average computer user.